boards/itsybitsy_m0/src/lib.rs

```diff
@@ -97,6 +100,53 @@ define_pins!(
     pin usb_dp = a25,
 );
 
+impl Pins {
+    /// Split the device pins into subsets
+    pub fn split(self) -> Sets {
+        let dotstar = Dotstar {
+            ci: self.dotstar_ci,
+            di: self.dotstar_di,
+            nc: self.miso,
+        };
+
+        Sets {
+            dotstar,
+            port: self.port,
+        }
+    }
+}
+
+/// Sets of pins split apart by category
+pub struct Sets {
+    /// Dotstar (RGB LED) pins
+    pub dotstar: Dotstar,
+    /// Port
+    pub port: Port,
+}
+/// Dotstar pins
+pub struct Dotstar {
+    pub ci: Pa0<Input<Floating>>,
+    pub di: Pa1<Input<Floating>>,
+    pub nc: Pa12<Input<Floating>>,
+}
+
+impl Dotstar {
+    pub fn dotstar<T: CountDown + Periodic>(
```

Comment:
im a fan of the init nomenclature
https://github.com/atsamd-rs/atsamd/blob/master/boards/pygamer/src/pins.rs#L399

https://github.com/atsamd-rs/atsamd/blob/master/boards/pygamer/examples/neopixel_tilt.rs#L42

Reply:
I'll refactor and split `lib.rs` so that there's a `pins.rs` like the other boards too then.